### PR TITLE
More robust line-ending detection for Bundles

### PR DESCRIPTION
**Pull request by zack-shoylev**

### CODE REVIEW

**Comment by zack-shoylev** [comment]
Seems broken on windows without that fix.

File: core/src/main/java/org/jclouds/osgi/Bundles.java [hunk 1]
```diff
@@ -131,6 +131,6 @@ public T apply(Class<?> in) {
    }
 
    private static Iterable<String> splitOrEmptyAndClose(InputStream in) throws IOException {
-      return Splitter.on('\n').omitEmptyStrings().split(toStringAndClose(in));
+      return Splitter.onPattern("\r?\n").omitEmptyStrings().split(toStringAndClose(in));
```
Comment (nacx) [inline core/src/main/java/org/jclouds/osgi/Bundles.java]: Could you use `Splitter.on(System.lineSeparator())` instead?

Comment (zack-shoylev) [inline core/src/main/java/org/jclouds/osgi/Bundles.java]: Yep, but this should be better/safer. The reason for this is that the jclouds repo does not use system line endings - so it is remotely possible you could get a bug expecting a system line ending and getting a different one.

Comment (zack-shoylev) [inline core/src/main/java/org/jclouds/osgi/Bundles.java]: In fact, "remotely possible" - I have seen that case with jclouds before. The repo specifically has to use core.autocrlf = false on windows or tests fail, and the code usually splits new lines on \n. Which is ok - just does not seem to be enough in this specific case.

Comment (nacx) [inline core/src/main/java/org/jclouds/osgi/Bundles.java]: Fair enough :) lgtm!

**Comment by zack-shoylev** [comment]
merged

**Pull request closed without merging**